data/sql/updates/pending_db_world/rev_1560396171418694700.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1560396171418694700');
+
+UPDATE `gameobject_template_addon` SET `faction`='35' WHERE  `entry`=193070; -- Nexus Raid Platform now has faction 35, instead of 0 and is friendly to players.
+UPDATE `gameobject_template` SET `Data0`='1500000' WHERE  `entry`=193070; -- Nexus Raid Platform has a lot more hp in order not to break in case any siege damage is taken, initial value was 100. (double safety)
```

Comment:
Thanks for your contribution. Just a few notes:

- No need to wrap numbers in single quotes
- No need to use 2 statements, you can use one single `UPDATE` to change multiple attributes
- Please add an empty line at the end of the file

Example:

```sql
UPDATE `gameobject_template` SET  `faction`=35`, `Data0`=1500000 WHERE  `entry`=193070;
```